book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] In order to properly show the `has-many` relationship between book and rental, getters and setters for rentals should be present.